conference.js

```diff
@@ -109,6 +109,7 @@ import {
 } from './react/features/overlay';
 import { setSharedVideoStatus } from './react/features/shared-video';
 import { isButtonEnabled } from './react/features/toolbox';
+import { endpointMessageReceived } from './react/features/subtitles';
```

Comment:
I wonder if endpointMessageReceived should be located somewhere other than subtitles if the endpoint messages could be used for more than subtitles. I suppose it doesn't hurt to keep it in subtitles for now though.

Reply:
iirc voting/polls might use it in the future.